frontend/src/entry/overview/FormEditPage.tsx

```diff
@@ -17,9 +17,10 @@ import { getStepsData, getStepsFromSchema } from 'utils/formUtils'
 
 type FormEditPageProps = {
   entry: EntryInterface
+  readOnly?: boolean
```

Comment:
This prop looks unnecessary to me due to your conditional logic in `frontend/src/entry/overview/Overview.tsx`